hyperjumptech/monika
# fix tests not running

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. #473 

## How did you implement / how did you fix it  
1.  Rename the `test/utils` folder to `test/others` folder. For some reason if the name of folder contains `util`, the tests inside it will be ignored. So `test/utilities` doesn't work either. 

## How to test  
1. Change a test to make it fail. For example, in `test/others/is-valid-url.test.ts`, change `http://www.example.com` to `htt://www.example.com`
2. Run `npm run test`
3. The test should finish with error.
